web/src/routes/CategoriesPage.tsx

```diff
@@ -31,15 +31,14 @@ import { cmsApiBaseUrl } from '../constants/urls'
 import DateFormatterContext from '../contexts/DateFormatterContext'
 import useWindowDimensions from '../hooks/useWindowDimensions'
 import BreadcrumbModel from '../models/BreadcrumbModel'
-import { urlFromPath } from '../utils/stringUtils'
 
 const CATEGORY_NOT_FOUND_STATUS_CODE = 400
 
 const getBreadcrumb = (category: CategoryModel, cityName: string) => {
   const title = category.isRoot() ? cityName : category.title
   return new BreadcrumbModel({
     title,
-    link: urlFromPath(category.path),
+    link: category.path,
```

Comment:
This perhaps breaks the json ld creation (see `JsonLdBreadcrumbs`. It is used there for the `item` property. So you need `urlFromPath` there I'd say. Optional: I would rename `link` to `pathname` to make it clearer that it is not a full url.

Reply:
are you sure this break? i don't see any difference:
with urlFromPath:
![image](https://user-images.githubusercontent.com/37902063/187680997-2cde49b9-b57e-42bb-a22a-b49222372e4e.png)
without:
![image](https://user-images.githubusercontent.com/37902063/187681194-e483ca8c-e292-4213-bbeb-0ae1d16690b2.png)

Reply:
For me it differs (using yarn start):
with:
` {"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Testumgebung","item":"/testumgebung/en"},{"@type":"ListItem","position":2,"name":"Children, youth and family","item":"/testumgebung/en/children-youth-and-family"},{"@type":"ListItem","position":3,"name":"School for children from 6 to 18 years","item":"/testumgebung/en/children-youth-and-family/school-for-children-from-the-age-of-6-to-15"}]}`

without:
`{"@context":"https://schema.org","@type":"BreadcrumbList","itemListElement":[{"@type":"ListItem","position":1,"name":"Testumgebung","item":"http://localhost:9000/testumgebung/en"},{"@type":"ListItem","position":2,"name":"Children, youth and family","item":"http://localhost:9000/testumgebung/en/children-youth-and-family"},{"@type":"ListItem","position":3,"name":"School for children from 6 to 18 years","item":"http://localhost:9000/testumgebung/en/children-youth-and-family/school-for-children-from-the-age-of-6-to-15"}]}`

It would be nice if we could make the naming clear nevertheless (either `path` if it is a path or `url` if it is an url).

Reply:
alright i added `item: urlFromPath(breadcrumb.link),` even i couldn't reproduce.
Well i renamed link to pathname. Didn't understand if there is more to do here or it was just a general hint (path/url)